glide.lock

```diff
@@ -0,0 +1,52 @@
+hash: d576153bffcb3088cc901b219fcac5ff48dfe76de64f8eec66c7ad730ae6a3aa
```

Comment:
Glide is a package manager for Go. This file lists the versions for all dependencies. There are no version numbers in the Go universe, at least they are not used consistently, so most dependency versions are provided as the SHA1 of the commit to check out from their Git repo.